src/Company/utils.ts

```diff
@@ -0,0 +1,11 @@
+import type { CompanyName, LocationName } from "@enums";
+
+type LocationNameString = `${LocationName}`;
+type CompanyNameString = `${CompanyName}`;
+type CompanyNamesAreAllLocationNames = CompanyNameString extends LocationNameString ? true : false;
+const __companyNameCheck: CompanyNamesAreAllLocationNames = true;
```

Comment:
lol, someone deserved a medal for this xD

Reply:
I initially just had a top-level for loop that was manually checking that all CompanyName members are also LocationName members, but doing the check at a type level made a lot more sense, since it avoids imports and also we don't have to build the game to figure out there's an issue.

It is pretty jank looking though.